tests/test_bmi_core_combined.py

```
@@ -167,28 +167,16 @@ class TestBMICoreCoverage:
-        except TypeError:
-            pass
-        try:
-            return fn(group, lang=lang)  # type: ignore[misc]
-        except TypeError:
-            pass
-        try:
-            return fn(lang, group=group)  # type: ignore[misc]
-        except TypeError:
-            pytest.skip("interpret_group signature unsupported for this test")
+            return fn(bmi=24.9, group=group, lang=lang)  # type: ignore[misc]
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)